client/blocks/subscription-length-picker/option.jsx

```diff
@@ -0,0 +1,122 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import Badge from 'components/badge';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from '../../lib/plans/constants';
+
+let componentNumber = 0;
+
+export class SubscriptionLengthOption extends React.Component {
+	static propTypes = {
+		term: PropTypes.string.isRequired,
+		savePercent: PropTypes.number.isRequired,
+		price: PropTypes.string.isRequired,
+		pricePerMonth: PropTypes.string.isRequired,
+		checked: PropTypes.bool.isRequired,
+		value: PropTypes.any.isRequired,
+		onCheck: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		checked: false,
+		savePercent: 0,
+		onCheck: () => null,
+	};
+
+	constructor( props ) {
+		super( props );
+		this.htmlId = 'subscription-option-' + ++componentNumber;
+	}
+
+	render() {
+		const { savePercent, price, term, checked } = this.props;
+		const className = classnames( 'subscription-length-option', {
+			'subscription-length-option--active': checked,
```

Comment:
Hmm we'd probably use an `is-active` modifier right? https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

Reply:
good catch, thanks!